modules/lang-painless/src/main/java/org/elasticsearch/painless/lookup/PainlessLookupUtility.java

```diff
@@ -336,6 +336,13 @@ public static boolean isConstantType(Class<?> type) {
                type == String.class;
     }
 
+    /**
+     * Constructs a painless constructor key used to lookup painless constructors from a painless class.
+     */
+    public static int buildPainlessConstructorKey(int constructorArity) {
```

Comment:
Do we really need a method for this? Can't we just say the key is always the constructor arity?

Reply:
Changed this to be a String so error messages will read out <init>/arity instead of just arity so people will know this is a constructor rather than just be baffled by some random number.